app/screens/ScheduleScreen/ScheduleScreen.tsx

```diff
@@ -59,11 +59,11 @@ export const ScheduleScreen: React.FC<TabScreenProps<"Schedule">> = () => {
   )
 
   const hScrollRef = React.useRef(null)
-  const scheduleListRefs = React.useMemo(() => {
-    return Object.fromEntries(
+  const scheduleListRefs = React.useRef(
+    Object.fromEntries(
       schedules.map((s) => [s.date, React.createRef<FlashList<ScheduleCardProps>>()]),
-    )
-  }, [schedules])
+    ),
+  ).current
```

Comment:
Adding `schedules` as a dep here caused this issue. Since the `useScheduleScreenData()` is building the schedules array from a hard coded array of dates it shouldn't be necessary to update after we receive events from the API. To make this more obvious I'm switching to using `useRef` here so we know it only needs to be setup once and to prevent possible exhaustive deps linter errors.